pages/index.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="page">
+      <input :class="{ error: !requestName }" @keyup.enter="requestName ? validRequestName() : null" id="label" name="label" type="label" v-model="label">
```

Comment:
What's the use of `validRequestName()` ?
I meant, why you want to pattern match a `label` string on every keyup?